Guides/UPI/vSphere_terraform/README.md

```diff
@@ -127,7 +127,7 @@ Extract `openshift-install` tool (e.g. `oc adm release extract --command=openshi
 
 ## Build a Cluster
 
-1. Create an `install-config.yaml` in an empty folder.
+1. Create an `install-config.yaml`. Copy it in the folder `install-dir`. Create a backup of it because it gets deleted in the next step.
```

Comment:
Lets add a warning that 
* `install-dir` is arbitrary name, users should pick some meaningful name
* install dir should not be reused across installs as it contains hidden files

Reply:
I added the warning. But I hardcoded the folder name 'install-dir'. I see no reason making it customizable. Is that ok for you?

Reply:
Yeah, that looks fine (we don't have to describe all available options in the guide anyway)